docs/Tutorials/zot-registry-tutorial.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Tutorial distribute an OCI artifact to Zot Registry using ORAS CLI
```

Comment:
I wonder if this should just be called something along the lines of introduction to OCI registries and artifacts.

Reply:
I think we'll add more use cases so we might have to mention zot. If not, I agree, we can remove zot from the title.

Reply:
I'd also recommend that we make any guidance to be as solution/vendor agnostic as possible

Reply:
Agreed, the title can say "... to OCI registries" and in the following text, make `zot` an example so readers have something to follow along and try.

Reply:
Also the term "artifact" (over time) is loaded/controversial/confusing. I think you want to refer to: https://github.com/opencontainers/image-spec/pull/1068 and related corpus.

Reply:
Thank you, everyone! I have updated the title now and it does not have zot in it.

Reply:
@rchincha if I understood correctly, I should be using the term "OCI image" here, right?

Reply:
"images and artifacts"?

Reply:
Done 👍